R/status.R

```diff
@@ -1,7 +1,7 @@
 
 `_renv_status_running` <- FALSE
 
-#' Status
+#' Report differences between lockfile and project library
```

Comment:
This also reports on other things though, e.g. you can use it to check the cache status with `status(cache = TRUE)`